CHANGELOG-1.1.md

```diff
@@ -3,6 +3,8 @@
 ## Deprecations
 
 * Command line flag `-connection-timeout` is deprecated and has no effect.
+* Command line flag `--leader-election-identity` is deprecated and has no effect.
+* Command line argument `configmaps` for flag `--leader-election-type` is deprecated in favor of `--leader-election-type=leases`. The default is still `configmaps` for backwards compatibility reasons but will be switched to `leases` in a future release.
```

Comment:
I thought of another way to document this. What do you think of [this](https://github.com/kubernetes-csi/external-provisioner/pull/263/commits/80918f5746898d01e14af19163df158904ee15f1)?

Reply:
Looks much better, I'll update :)